script/django/setup.bat

```diff
@@ -0,0 +1,7 @@
+pushd django
```

Comment:
Use `pushd/popd` instead of `cd` as that will leave you in the django directory after the script has ran which could be confusing for the user